### ISSUE TITLE
Add Install & Usage information to the wiki

### ISSUE THREAD

**Issue by markplagge** [markplagge/NeMo#31]
We need to add usage information to the Wiki, for basic scenarios. 
There are no installation instructions on the wiki as well - A guide would be good.

**Comment by nwolfey21** [comment]
+1 
 - Would be nice to simple instruction set/guide to get from nothing to running identity matrix (or some other simple test) on 4 mpi processes.
 - Example:
    * git clone <NeMo https link>
    * mkdir <some build dir>
    * ccmake <path to cmakelist.txt>
    * ...
    * mpirun -n 4 <NeMo executable> <NeMo params> ...

**Comment by markplagge** [comment]
Do you think a docker image would be helpful for this? I use on on the Titan server for doing base install testing...

**Comment by nwolfey21** [comment]
Is a docker image something a non sudoer can install and setup? I'm thinking a docker image might be an alternate/additional route for install but I'm not sure it can/should replace a simple list/sequence of commands for installing.

**Comment by markplagge** [comment]
Yes, you are right - a docker image would not replace a list/sequence of commands for installing. 
And yes, a non root user can install and run docker images on a system with docker installed. I'll do both.
